elasticdl/python/ps/parameters.py

```diff
@@ -197,3 +197,24 @@ def to_model_pb(self):
             model_pb.embedding_table_info.append(embedding_info)
 
         return model_pb
+
+    def debug_info(self):
+        info = ""
+        total_size = 0
+        for e in self.embedding_params:
```

Comment:
Use a more specific name instead of 'e'?

Reply:
Done